site/app/templates/forum/ForumBar.twig

```diff
@@ -25,6 +25,8 @@
 	</script>
 {% endif %}
 
+{% set buttonCounter = 1  %}
```

Comment:
Let tabindex be 0 . If tabindex is greater than 0. It will not work as needed as after all tab is navigated  then we come across `Search`, etc buttons which is not intended

Reply:
Understood! Thanks for pointing out !:)